### ISSUE TITLE
Product import doesn't change `Enable Qty Increments` field

### ISSUE THREAD

**Issue by simpleadm** [magento/magento2#14351]
### Preconditions
1. Magento 2.2.2 (all magento 2 versions)
2. Option `Stores > Configuration > Catalog > Inventory > Enable Qty Increments` marked as 'Use Config Settings' (`No`)

### Steps to reproduce

1. Create simple product with 'Test Simple' `SKU`
2. Fill `Quantity`, `Stock Status` and required fields
3. Make sure that `Enable Qty Increments` ('Advanced Inventory') marked as `Use Config Settings`
4. Go to `System > Data transfer: Import > Import products`
5. Select Import Behavior: Add/Update
6. Use attached csv file

### Expected result

1. `Enable Qty Increments` field have value from the csv file (e.g. Yes)
2. `Qty Increments` field have value from the csv file (e.g. 2)
![image](https://image.prntscr.com/image/INwoMk_ORt6C2VXsrHwvrQ.png)

### Actual result

1. ` Enable Qty Increments` field wasn't changed and still marked as `Use Config Settings`
2. `Qty Increments` field wasn't changed as well
![image](https://image.prntscr.com/image/XWZLjC1eRwyHxi9tFxOglg.png)

## Example files
``` csv
"sku","qty","manage_stock","enable_qty_increments","qty_increments"
"Test Simple",88,"Use Config",1,2
```

Best regards,
Sergey

**Comment by magento-engcom-team** [comment]
@simpleadm, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by sanjay-wagento** [comment]
I am working on this at #dmcdindia

**Comment by simpleadm** [comment]
@sanjay-wagento ,
please check my PRs:
- #14352
- #14379
- #14380

**Comment by sanjay-wagento** [comment]
@magento-engcom-team , I cannot reproduce the issue in the latest 2.2-develop branch (Magento ver. 2.2.6-dev)

**Comment by hdnarola** [comment]
@simpleadm , I have gone through your issue and debug into code. When you import product and if you would like to change setting (Yes/No) of "Enable Qty Increments" field in product advanced inventory then you need to make sure that you define "use_config_enable_qty_inc" field in CSV file because if there is using "Config Setting" value Yes(1) ticked then you must need to define "use_config_enable_qty_inc" as 0 and make 1 as "enable_qty_increments" in CSV file. It should be worked. I have tried same and it's working perfectly.

For better understanding, Kindly check CSV fields at below with its values.
-------------
"sku","qty","manage_stock","enable_qty_increments","qty_increments","use_config_enable_qty_inc"
"Test Simple",88,"Use Config",1,2,0
-------------
Let me know if you have still confustion.

Thank you.
#dmcindia

**Comment by simpleadm** [comment]
@hdnarola ,
Products import have logic for handling **Use config** settings and it work perfect for most fields (except of `Enable Qty Increments`). So you don't need to add new `use_config_` columns to the csv files.
More technical info you can find in the PR #14352

**Comment by hdnarola** [comment]
@simpleadm , OK. Got it.

**Comment by sidolov** [comment]
Hi @simpleadm. Thank you for your report. 
The issue has been fixed in magento/magento2#14352 by @simpleadm in 2.3-develop branch
Related commit(s):
 - [1268692cd2037f3d5df7f458bd91934c4272978a](https://github.com/magento/magento2/commit/1268692cd2037f3d5df7f458bd91934c4272978a)
 - [16b8b2c206e06945f2aa884e454bf2a4d832ee68](https://github.com/magento/magento2/commit/16b8b2c206e06945f2aa884e454bf2a4d832ee68)
 - [cb83575fa1fae10f3737a81ea96d9e375b00ce4b](https://github.com/magento/magento2/commit/cb83575fa1fae10f3737a81ea96d9e375b00ce4b)

The fix will be available with the upcoming 2.3.0 release.

**Comment by VladimirZaets** [comment]
Hi @simpleadm. Thank you for your report. 
The issue has been fixed in magento/magento2#14380 by @simpleadm in 2.1-develop branch
Related commit(s):
 - [ead41d96302630687627b6ba84187fefffbcee25](https://github.com/magento/magento2/commit/ead41d96302630687627b6ba84187fefffbcee25)
 - [053fea21816b022dc5ed04845612594a1ce7a537](https://github.com/magento/magento2/commit/053fea21816b022dc5ed04845612594a1ce7a537)

The fix will be available with the upcoming 2.1.15 release.

**Comment by VladimirZaets** [comment]
Hi @simpleadm. Thank you for your report. 
The issue has been fixed in magento/magento2#14379 by @simpleadm in 2.2-develop branch
Related commit(s):
 - [54507a7909b4c9c4eac57ec3436a35e408c03c4d](https://github.com/magento/magento2/commit/54507a7909b4c9c4eac57ec3436a35e408c03c4d)
 - [637979f5959a687fd14d57e725496ed469f683ed](https://github.com/magento/magento2/commit/637979f5959a687fd14d57e725496ed469f683ed)

The fix will be available with the upcoming 2.2.6 release.
